src/__integration_tests__/final-status-io.integration.test.ts

```diff
@@ -0,0 +1,61 @@
+import "expect-puppeteer";
+
+describe("Check final status on IO mapping tests", () => {
+  /**
+     * Test input and configuration
+  */
+
+  const ECOMMERCE_FE_ESITO_PAGE = "http://localhost:1234/ecommerce-fe/esito#clientId=IO&sessionToken=test&transactionId=";
+
+  /**
+   * Add all mock flow. Reference to the flow defined into the checkout be mock
+   */
+  const mockTransactionIdsWithExpectedResultMap = new Map([
+    ["302054585254587560","0"],
+    ["302054585254587561","0"],
+    ["302054585254587562","1"],
+    ["302054585254587563","0"],
+    ["302054585254587564","1"],
+    ["302054585254587565","1"],
+    ["302054585254587566","1"],
+    ["302054585254587567","1"],
+    ["302054585254587568","1"],
+    ["302054585254587569","4"],
+    ["302054585254587570","8"],
+    ["302054585254587571","8"],
+    ["302054585254587572","1"],
+    ["302054585254587573","8"],
+    ["302054585254587574","1"],
+    ["302054585254587575","2"],
+    ["302054585254587576","17"],
+    ["302054585254587577","1"],
+    ["302054585254587578","0"],
+    ["302054585254587579","17"]
+  ]);
+
+
+  /**
+   * Increase default test timeout (80000ms)
+   * to support entire payment flow
+    */
+  jest.setTimeout(120000);
+  jest.retryTimes(0);
```

Comment:
```suggestion
  jest.retryTimes(3);
```
You could be more forgiving than this 😄  Give your tests more than one shot!

Reply:
done here a5b54a401b6afac22749bca4532df13de8d94daa